spec/rails_helper.rb

```diff
@@ -1,4 +1,4 @@
-if ENV['COVERAGE'].present?
+if !!ENV['COVERAGE']
```

Comment:
For reasons that remain a mystery, a new issue was the Rails framework wasn't loaded at this point, and therefore the `.present?` method did not exist, meaning every test failed!